components/brave_shields/browser/base_brave_shields_service.h

```diff
@@ -35,7 +35,10 @@ class BaseBraveShieldsService : public BraveComponent {
                                   blink::mojom::ResourceType resource_type,
                                   const std::string& tab_host,
                                   bool* did_match_exception,
-                                  std::string* mock_data_url);
+                                  bool* did_match_important,
+                                  std::string* mock_data_url,
+                                  bool previously_matched_rule,
```

Comment:
input parameters should precede output ones

https://google.github.io/styleguide/cppguide.html#Inputs_and_Outputs

Reply:
Updated in https://github.com/brave/adblock-rust-ffi/pull/31, will fix `DEPS` here once that's merged